File: fizzbuzz.rs
fn main() {
    for x in 1..101 {
        let div_by_three = x % 3 == 0;
        let div_by_five = x % 5 == 0;
        
        match x {
            _fb if div_by_three && div_by_five => println!("FizzBuzz"),
            _f if div_by_three => println!("Fizz"),
            _b if div_by_five => println!("Buzz"),
            _ => println!("{}", x),
        }
    }
}

File: fizzbuzz.dart
void main() {
    for(int i = 1; i < 101; i++) {
        bool divByThree = i % 3 == 0;
        bool divByFive = i % 5 == 0;
        if (divByThree && divByFive) {
            print("FizzBuzz");
        } else if (divByThree) {
            print("Fizz");
        } else if (divByFive) {
            print("Buzz");
        } else {
            print(i);
        }
    }
}